utils/tests/format.test.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
```

Comment:
imo i would put this file in the same directory as `utils/format`, since its already differentiated as `format.test`

Reply:
It's generally good practice, given how many JS files there are already in this folder, to group the tests together so it's a bit cleaner in our code editor.